internal/character/danhengimbibitorlunae/skill.go

```diff
@@ -0,0 +1,88 @@
+package danhengimbibitorlunae
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	SkillEffect  = "danhengimbibitorlunae-skill-effect" // MAvatar_DanHengIL_00_Skill02_CriticalDamage
+	EnhanceLevel = "danhengimbibitorlunae-enhancelevel"
+)
+
+type skillState struct {
+	CritDMGBoost float64
+}
+
+// enhance attack,has 3 type,use 1/2/3 skill point
+
+func (c *char) initSkill() {
+	modifier.Register(EnhanceLevel, modifier.Config{
```

Comment:
Keeping track of the `EnhanceLevel` via a modifier is not really needed. 

> so give char some like skillCount variable that you just increment every time skill is used and then set to 0 in Attack 
-Kyle